src/style.scss

```diff
@@ -25,6 +25,18 @@
     --plotweaver-color-invest-front: #3e6abb;
     --plotweaver-color-invest-back: #182845;
 
+    // Defaults main colours:
+    --plotweaver-color-accent: #ffcb74;
+    --plotweaver-color-faded: #f6f6f6;
+    --plotweaver-color-sheet: #111111;
+
+    --plotweaver-color-base-1: #000000;
+    --plotweaver-color-base-2: #1f1f1f;
+    --plotweaver-color-base-3: #2f2f2f;
+    --plotweaver-color-base-4: #3f3f3f;
+    --plotweaver-color-base-5: #4f4f4f;
+    --plotweaver-color-base-6: #5f5f5f;
```

Comment:
There's already a `&.theme-plotweaver` as the default theme, why add a whole new set?

Reply:
To put it very simply, it doesn't work. I have attached an image to show how it looks if I comment out this section.
![image](https://github.com/user-attachments/assets/46304885-faea-4a71-a01f-13b1f5c52642)

Reply:
I suspect that's because we may have some dodgy scss syntax? though I feel like it should work.
Either way we'd need to make sure the theme class is applied to the correct element, which I'm not sure we do at the moment? It may be hard coded to something right now for dev though, so have a look. Might want to look at how various other modules handle theme switching? especially tying it to a world setting (I'll try looking up some of the Star Wars UI ones in this regard tomorrow and link back).
For now, this link might help? https://bcodes.io/blog/post/theming-css-variables-in-scss

Reply:
If the default theme doesn't work then the stormlight theme is unlikely to either, so I would figure out getting it to work before we commit anything since themes are very important to how the premium modules will be set up

Reply:
@zithith it is not to do with the CSS structure, but to how the root is not receiving the theme tag. Furthermore, @MangoFVTT, I might have to update this PR to not include issue #71 as I unfortunately do not have the time to debug the theming issue.
My current priority is making sure that the SVGs, colours and positions are correct.

Reply:
@PilotKayer Yeah there's definitely something to look into here, and I think it relates to the fact we are relying on default foundry dark mode to handle a lot for us in the base sheets.
In release 0.1.0 I added in a setting toggle for item sheet tabs that required inserting a class higher up the chain than the sheets themselves could handle, so we probably need to do something similar for the theme switch. Happy to pick up #71 myself and try this approach (after we've figured out what glaring parts of the 0.1 release need immediate attention)